src/GraphQL/Utilities/MetadataProvider.cs

```diff
@@ -10,20 +12,20 @@ namespace GraphQL.Utilities
     /// </summary>
     public class MetadataProvider : IProvideMetadata
     {
-        private Dictionary<string, object> _metadata;
+        private Dictionary<string, object>? _metadata;
```

Comment:
Maybe ` private Dictionary<string, object?>? _metadata;`, not sure.

Reply:
Already changed in a subsequent PR.